uproot/source/compressed.py

```diff
@@ -80,6 +82,13 @@ def decompress(self, source, cursor, compressedbytes, uncompressedbytes=None):
                 raise ValueError("lz4 needs to know the uncompressed number of bytes")
             return lz4_decompress(cursor.bytes(source, compressedbytes), uncompressed_size=uncompressedbytes)
 
+        elif self.algo == uproot.const.kZSTD:
+            try:
+                from zstd import decompress as zstd_decompress
+            except ImportError:
+                raise ImportError("Install zstd package with:\n    pip install zstd\nor\n    conda install -c anaconda zstd")
```

Comment:
The zstd package is in the `conda-forge` channel, not `anaconda`, and I just tried installing it: it appears to be the binary library (libstd.so), not the Python bindings. I think users will be forced to install the Python bindings through pip.

@chrisburr, do you happen to know the status of the zstd Python bindings in conda-forge? Is it coming, does it exist under a different name, or is it just not available?

Reply:
Looks like there missing but it should be trivial to add. I'll do it later today as `python-zstd`.

I would also suggest editing the `lz4` message above to suggest `conda-forge` instead of `anaconda` (or not suggest a channel at all).

Reply:
I think that's a good idea: removing the `-c conda-forge` from the lz4 and lzma messages (although the lzma message is only triggered in Python 2.7, so that's becoming legacy). If the user uses conda to install packages, they must have already added conda-forge as a channel, otherwise they would not have gotten uproot. The [uproot front-page](https://github.com/scikit-hep/uproot#installation) recommends adding conda-forge as a channel.

Reply:
> The zstd package is in the `conda-forge` channel, not `anaconda`, and I just tried installing it: it appears to be the binary library (libstd.so), not the Python bindings. I think users will be forced to install the Python bindings through pip.
> 
> @chrisburr, do you happen to know the status of the zstd Python bindings in conda-forge? Is it coming, does it exist under a different name, or is it just not available?

@chrisburr  @jpivarski thanks a lot for an input! I will fix PR accordingly.